pkg/image/anaconda_container_installer.go

```diff
@@ -58,6 +58,8 @@ type AnacondaContainerInstaller struct {
 	// Locale for the installer. This should be set to the same locale as the
 	// ISO OS payload, if known.
 	Locale string
+
+	InstallRootfsType string
```

Comment:
While I understand where this name comes from it feels slightly confusing with the other `RootfsType` in this struct. I don't have a better idea for a name myself though.

Reply:
It should have a comment that clearly states which rootfs it applies to.